apps/system/js/app_window.js

```diff
@@ -765,8 +765,10 @@
       this.browserContainer.appendChild(this.browser.element);
       this.iframe = this.browser.element;
       this.launchTime = Date.now();
+      // Register audio channels for new browser frame.
+      this._unregisterAudioChannels();
+      this._registerAudioChannels();
```

Comment:
Don't know why we get the below error message
```
E/GeckoConsole(15579): [JavaScript Error: "TypeError: this.app is null" {file: "app://system.gaiamobile.org/js/app_chrome.js" line: 488}]
```
if we do
```
this.uninstallSubComponents();
this.installSubComponents();
```
to instead of
```
this._unregisterAudioChannels();
this._registerAudioChannels();
```